apache-jena-osgi/jena-osgi/src/main/java/org/apache/jena/osgi/Activator.java

```diff
@@ -0,0 +1,47 @@
+package org.apache.jena.osgi;
+
+import org.apache.jena.system.JenaSubsystemRegistry;
+import org.apache.jena.system.JenaSubsystemRegistryBasic;
+import org.apache.jena.system.JenaSystem;
+import org.osgi.framework.BundleActivator;
+import org.osgi.framework.BundleContext;
+
+public class Activator implements BundleActivator {
+	
+	/* 
+	 * Based on http://svn.apache.org/repos/asf/aries/trunk/spi-fly/spi-fly-examples/spi-fly-example-provider-consumer-bundle/src/main/java/org/apache/aries/spifly/pc/bundle/Activator.java
+	 * the Activator#start() waits for bundle extension by Aries SPI Fly, configures JenaSystem logging and requests for initialization.     
+	 */
+	public void start(BundleContext context) throws Exception {
+
+		Thread t = new Thread(new Runnable() {
+			@Override
+			public void run() {
+				try {
+					Thread.sleep(500);
+				} catch (InterruptedException e) {
+				}				
+				setUpJena();
+			}
+		});
+		t.start();
+	}
+
+	private void setUpJena() {
+		JenaSubsystemRegistry r = new JenaSubsystemRegistryBasic() {
+			@Override
+			public void load() {
+				super.load();
+			}
+		};
```

Comment:
Sure, I've been down enough roads with the package fandango to know it's just part of the journey. My suggestion is just towards documentation, because we would be making a change that would require action on the part of downstream folks. I think it would be great to offer provisioning artifacts, although I'm not sure they belong as a part of Jena itself (I'll shut up and let @afs and other committers speak to that, I'm just an interested contributor). If that does become part of the picture, inside Jena or next to it, there is previous work available in connection with Fedora Commons (for which I am a committer) that could be useful (Karaf features, actually).